python-sqlite-sqlalchemy/README.md

```diff
@@ -1,20 +1,27 @@
 # Python / Sqlite /SqlAlchemy article
 
-This repository contains the content and example code 
-for the python/sqlite/sqlaclchemy article I'm writing 
+This repository contains the content and example code
+for the python / sqlite / sqlaclchemy article I'm writing
 for Real Python.
 
 This project was built using Python 3.8.0
 
+## Python Virtualenv
+
+I use the `pyenv` tool to install Python versions on my Mac. I find it a very useful tool, and the instructions that follow use it, and are based on having Python version 3.8.0 installed using the following command:
+
+```shell
+pyenv install 3.8.0
```

Comment:
I think we should use a `$` prefix for console commands, both here and in the bullet points below:

```suggestion
$ pyenv install 3.8.0
```